cpg_utils/hail_batch.py

```diff
@@ -101,7 +85,7 @@ def copy_common_env(job: hb.batch.job.Job) -> None:
     """
     # If possible, please don't add new environment variables here, but instead add
     # config variables.
-    for key in ('CPG_CONFIG_PATH',):
+    for key in 'CPG_CONFIG_PATH':
```

Comment:
This will iterate over the string, as it's not a tuple anymore.

Reply:
Good catch, thanks!